opaque_keys/edx/locator.py

```diff
@@ -1620,3 +1621,59 @@ def html_id(self) -> str:
         # HTML5 allows ID values to contain any characters at all other than spaces.
         # These key types don't allow spaces either, so no transform is needed.
         return str(self)
+
+
+class LibraryCollectionLocator(CheckFieldMixin, LibraryCollectionKey):
+    """
+    When serialized, these keys look like:
+        lib-collection:org:lib:collection-id
+    """
+    CANONICAL_NAMESPACE = 'lib-collection'
+    KEY_FIELDS = ('org', 'lib', 'usage_id')
```

Comment:
Based on [this example](https://github.com/openedx/opaque-keys/blob/e3ff649fb59dfbc316fb99f33e3fed57e36f8201/opaque_keys/edx/locator.py#L649) I think this should be:

```suggestion
    KEY_FIELDS = ('lib_key', 'usage_id')
```

Or `('lib_key', 'collection_id')` - see below.

Reply:
Updated: https://github.com/openedx/opaque-keys/pull/335/commits/168c373f4adafe7c81f707baef3a971f7404acd7